sdk/storage/Azure.Storage.DataMovement/tests/LocalFileStorageResourceTests.cs

```diff
@@ -82,60 +81,79 @@ public void Ctor_string()
         }
 
         [Test]
-        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/32858")]
         public async Task ReadStreamAsync()
         {
             // Arrange
             var size = Constants.KB;
             string path = await CreateRandomFileAsync(Path.GetTempPath(), size:0);
-            var data = GetRandomBuffer(size);
-            using (var stream = new MemoryStream(data))
+            try
             {
-                using FileStream fileStream = new FileStream(path, FileMode.OpenOrCreate);
-                await stream.CopyToAsync(fileStream);
-            }
+                var data = GetRandomBuffer(size);
+                using (var stream = new MemoryStream(data))
```

Comment:
Not sure the type returned by GetRandomBuffer but I _think_ you could consider using `File.WriteAllBytes()` to write the data to the local file instead of using a `MemoryStream` and copying it?

Just a thought as its probably doing the same thing. And if this is an established pattern in the file/package, then also fine to leave it for that reason.

Reply:
Also see this is only part of the diff because of indent so totally fine to leave it :)